lsp-completion.el

```diff
@@ -561,6 +548,16 @@ Returns resolved completion item details."
          (string-lessp label-left label-right)
        (string-lessp sort-text-left sort-text-right)))))
 
+(defun lsp-completion--make-items (items markers prefix)
```

Comment:
This function is misleading as we do the filtering here so it would be better to spell it out.
Or we can keep the previous `lsp-completion--filter-candidates` function. How about change `markers` and `prefix` to a combined value of `metadata` instead? Would it make the function clearer?

Reply:
deleted